<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_text_dialog</class>
 <widget class="QDialog" name="add_text_dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Text Window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_text">
     <property name="text">
      <string>Your text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_color">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_color">
     <property name="text">
      <string>Text color</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Font:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFontComboBox" name="fontComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_font">
     <property name="text">
      <string>Font size:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBox_fontsize">
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>36</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Position:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="spinBox_X">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="prefix">
        <string>x:</string>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_Y">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="prefix">
        <string>y:</string>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_save">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>SAVE</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
